.eslintrc.json

```diff
@@ -12,7 +12,7 @@
     "jsx-quotes": ["error", "prefer-double"],
     "quotes": ["error", "single"],
     "no-multiple-empty-lines": [2, { "max": 2, "maxEOF": 1 } ],
-    "max-len": ["error", { "code": 120 }],
+    "max-len": ["error", { "code": 140 }],
```

Comment:
It's not good to edit ESLint rules on the go. Much better to add comment like
```
// eslint-disable-next-line max-len
```
before the line that causes trouble (only if it is _impossible_ to refactor it to be **<120 symbols**)